content/blog/2019/12/2019-12-31-happy-new-year.adoc

```diff
@@ -0,0 +1,99 @@
+More than **50** plugins have been also updated in order to offer better configuration-as-code support.
+As a result, the JCasC Plugin got massive adoption this year (from **2000** to almost **8000** installations),
+and now it becomes a de-facto standard for managing Jenkins as code.
+This year we also ran out very first CommunityBridge project devoted to link:/projects/jcasc/dev-tools/[JCasC Schema validation and developer tools].
+
+The link:/security/[Jenkins Security] was pretty hot this year.
+There were **5** link:/security/advisories/[security advisories] for the core and *20* - for plugins.
+In total we disclosed 288 security vulnerabilities across the project, including some backlog cleaning for unmaintained plugins.
+
+There were **2605** plugin releases, and **>215** NEW plugins have been hosted in the Update Center.
+In particular Jenkins ecosystem has greatly expanded into the Cloud space
+by offering dozens of new plugins (e.g. for Azure and Kubernetes).
+We also got many new plugins providing integrations with various Development and DevOps tools.
+
+So, keep updating Jenkins nd exploring new features!
+
+## What's next?
+
+Year 2020 will be pretty busy for the Jenkins project.
+There are many long-overdue changes in the project, which need to happen if we want the project to succeed.
+As it was written link:/blog/2019/12/16/board-election-results/#whats-next-for-the-board[Board elections blogpost],
+there are many areas to consider: UX revamp, cloud native Jenkins, pluggable storage, etc.
+In the coming months there will be a lot of discussions in link:/mailing-lists/[mailing lists] and link:/sigs/[special interest groups],
+and we invite all teams to work on their roadmaps and to communicate them in the community.
+
+Next month we will participate in link:https://fosdem.org/2020/[FOSDEM], and there will be a Jenkins stand there.
+On January 31st we will also host a traditional contributor summit in Brussels,
+where we will talk about next steps for the project, in terms of technical roadmaps and the project governance.
+If you are interested in Jenkins, stop by at our community booths and us at the summit!
+See link:https://groups.google.com/forum/#!topic/jenkinsci-dev/PUgV5xxHMwgp[this thread] for more information.
+
+We also plan to continue all outreach programs.
+At the moment we are looking for Google Summer of Code 2020 mentors and project ideas (link:/blog/2019/12/20/call-for-mentors/[announcement]),
+and we will be also interested to consider non-coding projects as a part of other programs like CommunityBridge.
+We also work on improving link:/participate/[contribution guidelines] for newcomers and expert contributors.
+If you are interested, please contact the link:/sigs/advocacy-and-outreach/[Advocacy and Outreach SIG].
+
+
+If you are interested in Jenkins, stop by at our community booths and join the contributor summits/hackathons.
+We also want to participate in Google Summer of Code 2018,
```

Comment:
This seems like a repeat of the sentence that was written in the preceding paragraph.  Should it be removed?
```suggestion
We also want to participate in Google Summer of Code 2020,
```